reports.py

```diff
@@ -412,7 +412,7 @@ def publish_report( self, title, content ):
 		report_title = dict_obj[ str( title ) ]
 		print str( reports_base_url + report_title )
 		page = self.site.Pages[ reports_base_url + report_title ]
-		page.save( content, summary = 'Updating report' )
+		page.save( content, summary = dict_obj[ 'summary' ] )
```

Comment:
Seems logical. We should add the key to other language dicts as well but I can do that outside of this patch.